### PR TITLE
Missing colon

### PR DESCRIPTION
**Pull request by stevealliance**
Missing colon generate an http request on a secure page.
The side effect was: notification was blocked by the browser and giving us 30% of Google numbers.

## Type of Change

- [ ] New adapter
- [ ] Feature Update
- [x ] Bug Fix
- [ ] Code Refactor
- [ ] Other

### CODE REVIEW

**Comment by ix-certification** [comment]
System tests are failing when using https. Now that the secure property is being set properly, the tests should not be expecting secure to always equal 0 in the validateBidRequest function.

**Comment by stevealliance** [comment]
Just update and run the system test all tests passed

File: districtm-dmx/districtm-dmx-htb.js [hunk 1]
```diff
@@ -154,7 +154,7 @@ function DistrictmDmxHtb(configs) {
             obj.id = System.generateUniqueId();
             obj.tagid = String(tag.xSlotRef.dmxid);
             tag.xSlotRef.id = obj.id;
-            obj.secure = window.location.protocol === 'https' ? 1 : 0;
+            obj.secure = 1;
```
Comment (ix-certification) [inline districtm-dmx/districtm-dmx-htb.js]: Just to confirm, the intent is to force the use of secure protocol in all scenarios?

Comment (stevealliance) [reply districtm-dmx/districtm-dmx-htb.js]: Yes, we have the same change pending for prebid 3.0

**Pull request merged**